templates/projects/view.tmpl

```diff
@@ -67,7 +67,7 @@
 	<div class="board {{if .CanWriteProjects}}sortable{{end}}">
 		{{range .Columns}}
 			<div class="ui segment project-column" style="background: {{.Color}} !important;" data-id="{{.ID}}" data-sorting="{{.Sorting}}" data-url="{{$.Link}}/{{.ID}}">
-				<div class="project-column-header">
+				<div class="project-column-header{{if and $canWriteProject (ne .ID 0)}}{{/* ID 0 is default column which cannot be moved */}} tw-cursor-grab{{end}}">
```

Comment:
Condition won't be needed with https://github.com/go-gitea/gitea/pull/29874 anymore.

Reply:
Yes, that's the only "interaction" it has and it would cause a merge conflict anyways.

Reply:
Conflict fixed.